source/index.html.md

```diff
@@ -23,6 +23,8 @@ includes:
   - administration/identity_providers
   - administration/service_providers
   - administration/saml_settings
+  - administration/branding
+  - administration/knowledge_base
```

Comment:
Was I supposed to add this as well when adding the API docs for the notifications and categories?

Reply:
yep... otherwise it is not shown in the menu

Reply:
Opened a PR for it, it will be merged shortly